addons/web_editor/tools.py

```diff
@@ -0,0 +1,132 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import json
+import re
+import requests
+import odoo
+
+from odoo import _
+from odoo.tools import image_process
+from odoo.modules.module import get_module_resource
+
+
+def get_video_platform_and_id(video_url):
```

Comment:
The third value (match) is required on youtube only. We just returned it for all the video providers for consistency (in case needed in the future). We can remove it for other providers than youtube if you want.

Reply:
no I have no problem with it. And I prefer to have a function that always return same type / same length.

I just mean, by this comment, to have docstring and explicit method name.
No problem with the code, just the name that less suppose that we will have 2 values and not 3

Reply:
Done! 👍